cpp/core/jni/JniWrapper.cc

```diff
@@ -305,9 +305,11 @@ void JNI_OnUnload(JavaVM* vm, void* reserved) {
 
 JNIEXPORT jlong JNICALL Java_io_glutenproject_exec_ExecutionCtxJniWrapper_createExecutionCtx( // NOLINT
     JNIEnv* env,
-    jclass) {
+    jclass,
+    jbyteArray sessionConf) {
   JNI_METHOD_START
-  auto executionCtx = gluten::createExecutionCtx();
+  auto sparkConf = gluten::getConfMap(env, sessionConf);
+  auto executionCtx = gluten::ExecutionCtx::create(gluten::kVeloxExecutionCtxKind, sparkConf);
```

Comment:
Velox-specific const `gluten::kVeloxExecutionCtxKind` appeared in common module?